ansible/playbook.yml

```diff
@@ -0,0 +1,95 @@
+- name: setup servers
+  hosts: managed_node
+  remote_user: ec2-user
+  become: yes
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: "Trello API Key: > "
+      private: yes
+
+    - name: trello_api_token
+      prompt: "Trello API Token: > "
+      private: yes
+
+  tasks:
+    - name: Install Git & Python
+      yum:
+        name:
+          - git
+          - python3
+        state: present
+
+    - name: Install pip for Python 3
+      yum:
+        name: python3-pip
+        state: present
+
+    - name: Create todoapp user
+      user:
+        name: todoapp
+        state: present
+        shell: /bin/bash
+
+    - name: Set POETRY_HOME environment variable for todoapp user
+      shell: echo "export POETRY_HOME=/home/todoapp/.local" >> /home/todoapp/.bashrc
+      args:
+        creates: /home/todoapp/.local
+
+    - name: Install Poetry for todoapp user
+      shell: >
+        curl -sSL https://install.python-poetry.org | POETRY_HOME=/home/todoapp/.local python3 -
+      args:
+        creates: /home/todoapp/.local/bin/poetry
+      become: yes
+      become_user: todoapp
+
+    - name: Create /opt/todoapp directory
+      file:
+        path: /opt/todoapp
+        state: directory
+        mode: '0755'
+      become: yes
+      become_user: ec2-user
```

Comment:
I'm not sure I follow why this is done as `ec2-user` rather than `todoapp` directly? Does `todoapp` miss some permissions initially?
Edit: I see that `todoapp` actually has these perms, so I'm guessing this is just a mismatch between your latest version and the PR!